crates/bevy_pbr/src/ssao/mod.rs

```diff
@@ -153,12 +153,35 @@ pub struct ScreenSpaceAmbientOcclusionBundle {
 /// TAA ([`bevy_core_pipeline::experimental::taa::TemporalAntiAliasing`]).
 /// Doing so greatly reduces SSAO noise.
 ///
-/// SSAO is not supported on `WebGL2`, and is not currently supported on `WebGPU` or `DirectX12`.
-#[derive(Component, ExtractComponent, Reflect, PartialEq, Eq, Hash, Clone, Default, Debug)]
-#[reflect(Component, Debug, Default, Hash, PartialEq)]
-#[doc(alias = "Ssao")]
+/// SSAO is not supported on `WebGL2`, and is not currently supported on `WebGPU`.
+#[derive(Component, ExtractComponent, Reflect, PartialEq, Clone, Debug)]
+#[reflect(Component)]
 pub struct ScreenSpaceAmbientOcclusion {
+    /// Quality of the SSAO effect.
     pub quality_level: ScreenSpaceAmbientOcclusionQualityLevel,
+    /// The constant thickness of objects use for the SSAO effect.
+    ///
+    /// This value is used to decide how far behind an object a ray of light needs to be in order
+    /// to pass behind it. Any ray closer than that will be occluded.
+    pub constant_object_thickness: f32,
+}
+
+impl Default for ScreenSpaceAmbientOcclusion {
+    fn default() -> Self {
+        Self {
+            quality_level: ScreenSpaceAmbientOcclusionQualityLevel::default(),
+            constant_object_thickness: 0.25,
+        }
+    }
+}
+
+impl Eq for ScreenSpaceAmbientOcclusion {}
+
+impl hash::Hash for ScreenSpaceAmbientOcclusion {
```

Comment:
Do we still need this Hash/Eq implementation? Better to remove it if not imo.

Reply:
My understanding was that, since this is a user-facing type, Hash/Eq might be useful.

Reply:
I've removed the implementation and change the pipeline key to only use the quality level instead of the whole struct.